modules/tinymce/src/themes/silver/main/ts/api/Settings.ts

```diff
@@ -118,6 +116,25 @@ const isStickyToolbar = (editor: Editor) => {
 
 const isDraggableModal = (editor: Editor): boolean => editor.getParam('draggable_modal', false, 'boolean');
 
+const getMenus = (editor: Editor) => {
+  const menu = editor.getParam('menu');
+
+  if (menu) {
+    return Obj.map(menu, (menu) => ({ ...menu, items: menu.items }));
+  } else {
+    return {};
+  }
+};
+
+const getMenubar = (editor: Editor) => editor.getParam('menubar');
+
+export interface ToolbarGroupSetting {
```

Comment:
You've duplicated the `ToolbarGroupSetting` interface, which is not a good idea as it's likely we'll forget to update both. Please ensure there are only one of these types.

Reply:
How would you like me to do that? I'm guessing putting the interface in its own file isn't a great idea, so how would we use it in two files at once?

Reply:
The `ToolbarGroupSetting` type would likely make sense here, as that's where it primarily applies to and then we can just import it from `Settings` in `Render.ts`.